lib/utils/cloud.ts

```diff
@@ -150,10 +148,17 @@ export async function downloadOSImage(
 	const manager = await import('balena-image-manager');
 	const stream = await manager.get(deviceType, OSVersion);
 
+	const displayVersion = await new Promise((resolve, reject) => {
+		stream.on('error', reject);
+		stream.on('version', resolve);
+	});
+
 	const visuals = getVisuals();
-	const bar = new visuals.Progress(`Downloading Device OS${displayVersion}`);
+	const bar = new visuals.Progress(
+		`Downloading balenaOS version ${displayVersion}`,
```

Comment:
Is it always correct to say `balenaOS` here, instead of `Device OS`?
Sample output after these changes:
```
Downloading balenaOS version 2.83.21+rev1.prod [========                ] 34% eta 30s
```
Whereas previously it might have read:
```
Downloading Device OS recommended [=                       ] 3% eta 1m17s
```